psamm/moma.py

```diff
@@ -79,84 +100,332 @@ def __init__(self, model, solver):
             mass_balance.add(lhs == 0)
 
     @property
+    # Returns the problem that we are working on
     def prob(self):
+        """Returns the underlying LP problem."""
         return self._prob
 
+    # Creates a constraints object
     def constraints(self, *args):
+        """Returns a constraint object that is used to constrain the model
+        for solving."""
         return ConstraintGroup(self, *args)
 
+    # Returns a generator of all the non exchange reactions in the model
     def _adjustment_reactions(self):
+        """Returns a generator of all the non exchange reactions
+        in the model."""
         for reaction_id in self._model.reactions:
             if not self._model.is_exchange(reaction_id):
                 yield reaction_id
 
+    # Returns a generator of all the reactions ids
+    def _all_reactions(self):
+        """Reaturns a generator of all the reactions in the model."""
```

Comment:
typo: "Returns a generator...". Actually PEP 257 (https://www.python.org/dev/peps/pep-0257/) recommends a style where the initial verb is imperative i.e. ("Return a ..." instead of "Returns a ..."). I don't care too much about this recommendation but if you want to get into the habit of following this style, you can change these.